.github/workflows/javascript.yml

```diff
@@ -0,0 +1,27 @@
+name: JavaScript
+
+on:
+  push:
+    branches: ['main']
+    paths: ['codes/javascript/**/*.ts']
+  pull_request:
+    branches: ['main']
+    paths: ['codes/javascript/**/*.ts']
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20.x
+      - uses: denoland/setup-deno@v1
```

Comment:
Do we need deno for JS?

Reply:
Yes, `test_all.js` is executed through Deno.